src/main/java/io/confluent/connect/jdbc/source/JdbcSourceTask.java

```diff
@@ -216,6 +235,24 @@ public void start(Map<String, String> properties) {
     running.set(true);
   }
 
+  private List<Map<String, String>> possibleTablePartitions(String table) {
+    TableId tableId = dialect.parseTableIdentifier(table);
+
```

Comment:
How about adding a comment:

    // Define source partition map for offset protocol V1 that uses the table FQN